src/generators/indexGenerator.ts

```diff
@@ -36,6 +36,7 @@ export function generateIndexFile(
 function generateRLCIndexForMultiClient(file: SourceFile) {
   const { model } = getSession();
   const clientName = model.language.default.name;
+  const createClientFuncName = `create${normalizeName(clientName, NameType.Class)}`;
```

Comment:
I would like to keep the factory function name as we had before but without the "Client" suffix. For example `PurviewCatalogClient` -> `PurviewCatalog` (not createPurviewCatalog). It is a common convention to have these factory functions following the pascal casing similar to calling a constructor.

Reply:
If there is a collision with the interface name we could name the interface `PurviewCatalogLike` which seems to be a common convention in TypeScript

Reply:
okay changed.